resources/views/support/components/components-repeater.blade.php

```diff
@@ -1,6 +1,12 @@
 @php
     if(!empty($page['page_components'])) {
         $components = $page['page_components'];
+
+
+        foreach ($page['page_components'] as $component) {
+            if (!empty($component['image_gallery'])) {
+            }
```

Comment:
Hi @ostafinskim 

It looks like this if statement doesn't do anything? Do you know if it serves some sort of purpose that I'm missing?

If not, could it please be removed?